src/common/swagger-options.ts

```diff
@@ -0,0 +1,61 @@
+import { ERROR_MESSAGES } from './errors/errors-messages'
+import { MessageOutputDto } from '../modules/auth/dtos/message/message.output.dto'
+
+export const SWAGGER_OPTIONS = {
+  server: {
+    internalServerError: {
```

Comment:
you don't need to describe default errors here. you need to describe a data structure of error that will throw your server and then put to the appropriate decorator like ApiForbiddenResponse or ApiInternalServerErrorResponse

Reply:
something like that 
```
{
  type: string //  = VALIDATION_ERRORS or BUSINESS_ERRORS
  errors: Errors[]
  data?: any
}
```